client/my-sites/checklist/wpcom-checklist/index.jsx

```diff
@@ -46,7 +46,7 @@ const getTaskList = memoize(
 class WpcomChecklist extends PureComponent {
 	static propTypes = {
 		createNotice: PropTypes.func.isRequired,
-		designType: PropTypes.oneOf( [ 'blog', 'page', 'portfolio' ] ),
+		designType: PropTypes.oneOf( [ 'blog', 'page', 'portfolio', 'store' ] ),
```

Comment:
I know we need this to get past the designType error but I don't think we need this. @michaeldcain any thoughts on this ? I'm not exactly sure what designType is for

Reply:
`designType` is technically used to route the user into the `store-nux` Signup flow. It's legacy code that classifies a site, and will be replaced by the `siteType` with the launch of the new Signup steps. This is fine as-is to get around the error.